bundles/plugins/org.bonitasoft.studio.connectors/src/org/bonitasoft/studio/connectors/configuration/AbstractConnectorBarResourceProvider.java

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright (C) 2012 BonitaSoft S.A.
```

Comment:
;even if it is more or less a code that has moved, we should put 2015  or at least 2012-2015 an dupdate BonitaSoft --> Boonitasoft and adress